lib/metalsmith-title-checker.js

```diff
@@ -36,6 +36,8 @@ function getDuplicateTitles (groupedFiles) {
       // Default examples always have a duplicated title,
       // this is OK since the iframes are appended with 'example'.
       .filter(file => !file['__filename'].includes('/default/'))
+      // Filter out Windows default paths too:
+      .filter(file => !file['__filename'].includes('\\default\\'))

```

Comment:
My main thinking was it only filters out matches, so seemed easier to add another even if it's redundant for Linux users and vice versa, instead of requiring the paths library and then using `path.normalize` - I'd be happy to change though